### PR TITLE
feat: i18n for undo messages and fix some missing

### PR DESCRIPTION
**Pull request by wafarm**
### Description/描述

Added i18n support for feedback messages. GithubAuthScreen wasn't changed.

### Related Issues/相关的 issue

 

Related to #29 

### Checklist/清单

- [x] The issues this PR addresses are related to each other/此 PR 解决的 issue 是相关的
- [x] My changes are all related to the related issue above/我的修改都与上方的相关 issue 有关
- [x] My changes generate no new warnings/我的修改没有产生新的警告
- [x] My code builds and runs on my machine/我的代码在我的电脑上可以编译和运行
- [x] I documented my code/我已经为我的代码添加了注释/文档

### CODE REVIEW

**Comment by Qubik65536** [comment]
Looks great so far, btw put this PR as draft if it’s not done yet.

**Comment by wafarm** [comment]
> Looks great so far, btw put this PR as draft if it’s not done yet.

It's already done. But I will add `GithubAuthScreen` if needed.

**Comment by zly2006** [comment]
> It's already done. But I will add `GithubAuthScreen` if needed.

Please dont do that. this screen should be rewrite with LibGUI later.

**Review by zly2006 — APPROVED** [review]

**Review by zly2006 — CHANGES_REQUESTED** [review]
Not complete. please fix following:

`com.github.zly2006.reden.mixin.chat.ChatScreenMixin`

`com.github.zly2006.reden.access.PlayerData.UndoRecord.Cause`

File: src/main/java/com/github/zly2006/reden/access/PlayerData.kt [hunk 1]
```diff
@@ -91,13 +91,13 @@ class PlayerData(
...
-            COMMAND(Text.of("Command")),
-            LITEMATICA_TASK(Text.of("Litematica Task")),
-            UNKNOWN(Text.of("Unknown"));
+            BREAK_BLOCK(Text.translatable("reden.feature.undo.break_block")),
+            USE_BLOCK(Text.translatable("reden.feature.undo.use_block")),
+            USE_ITEM(Text.translatable("reden.feature.undo.use_item")),
+            ATTACK_ENTITY(Text.translatable("reden.feature.undo.attack_entity")),
+            COMMAND(Text.translatable("reden.feature.undo.command")),
+            LITEMATICA_TASK(Text.translatable("reden.feature.undo.litematica_task")),
+            UNKNOWN(Text.translatable("reden.feature.undo.unknown"));
```
Comment (zly2006) [inline src/main/java/com/github/zly2006/reden/access/PlayerData.kt]: Suggest adding `cause` in your translation key, or just modify the key below.

**Review by zly2006 — CHANGES_REQUESTED** [review]

**Review by Qubik65536 — APPROVED** [review]
LGTM

**Review by zly2006 — APPROVED** [review]

**Comment by Qubik65536** [comment]
@zly2006 I do recommend you to do a beta release for this.

**Comment by zly2006** [comment]
You can do it yourself (

**Comment by Qubik65536** [comment]
> You can do it yourself (

Cool, I'll do it :)

**Pull request merged**